threedi_graph.py

```diff
@@ -77,23 +74,21 @@ def onCloseChildWidget(self, widget_nr):
         # close widget
         if nr is not None:
             widget = self.dock_widgets[nr]
-            widget.closingWidget.disconnect(self.onCloseChildWidget)
+            widget.closingWidget.disconnect(self.on_close_child_widget)
```

Comment:
is dat `widget.closingWidget` nog iets dat in `widget.closing_widget` hernoemd kan worden? (Kan zijn dat dat juist vanuit QT komt en dan daar er raar uitziet, hoor).